styleguide/source/_patterns/01-molecules/06-components/07-featured-content-hero-row/_featured-content-hero-row.scss

```diff
@@ -44,15 +44,13 @@
 
   @include breakpoint(740px min-width) {
     @include grid__unit--cols(5);
+    padding-right: 0;
+    
     img {
       float: right;
       max-height: 100%;
       max-width: 100%;
       width: initial;
     }
   }
-
-  @include breakpoint($bp-small) {
```

Comment:
@rpama please move the breakpoint declaration above the `img` styling - we should be grouping includes before nested elements.